client/app/dedicated/server/bandwidth/bandwidth.service.js

```diff
@@ -0,0 +1,75 @@
+class BaseDedicatedService {
+  constructor($q) {
+    this.$q = $q;
+  }
+
+  acceptResponse(data, message) {
+    return this.$q.resolve({
+      status: 'OK',
+      data,
+      message,
+    });
+  }
+
+  rejectResponse(data, message) {
+    return this.$q.reject({
+      status: 'ERROR',
```

Comment:
I'm having trouble understanding the point here, if you `$q.reject` you know you have an error because you'll go through `catch` (same applies for `resolve` and `then`), right ?

Reply:
This is an existing service that was moved.
But this file `bandwidth.service.js` is a duplicate I forgot to remove. The real file is dedicated-server-bandwidth-vrack-order.service.js.